target/linux/generic/config-6.6

```diff
@@ -3717,6 +3717,7 @@ CONFIG_MTD_MAP_BANK_WIDTH_4=y
 # CONFIG_MTD_NAND_ORION is not set
 # CONFIG_MTD_NAND_PASEMI is not set
 # CONFIG_MTD_NAND_PLATFORM is not set
+# CONFIG_MTD_NAND_RB91X is not set
```

Comment:
This symbol should be moved to `https://github.com/openwrt/openwrt/blob/main/target/linux/ath79/nand/config-default`. It's an ath79 downstream driver.

Reply:
I placed it here to match config-6.1 and config-5.15.

Reply:
Because `make kernel_oldconfig CONFIG_TARGET=subtarget` can automatically add it, there is no need to move it to the generic config. The fix has been merged just now https://github.com/openwrt/openwrt/pull/15712.

Reply:
Removed.